package.json

```diff
@@ -12,8 +12,7 @@
     "serve": "parcel index.html",
     "test": "jest",
     "test:watch": "jest --watch",
-    "watch:back": "parcel watch --target electron main.ts",
-    "watch:front": "parcel watch --no-hmr --public-url ./ index.html"
+    "dev": "node ./dev.js"
```

Comment:
I would rename `dev.js` file to `launchDevEnironment.js`

the "target" name `dev` feels fine though.